src/replicache.test.ts

```diff
@@ -468,10 +457,92 @@ test.skip('conflicting commits', async () => {
 });
 
 test('sync', async () => {
+  const diffServerURL = 'https://diff.com/pull';
+  const batchURL = 'https://batch.com';
+
+  let pullCounter = 0;
+  fetchMock.post(
+    diffServerURL,
+    () =>
+      [
+        {
+          stateID: '97dd36bqlpojn302g24hemq2o34v66qm',
```

Comment:
This is kind of brittle. I got the JSON responses by running agains the live servers and recorded the requests